Eatery/Controllers/BRB/BRBAccountManager.swift

```diff
@@ -0,0 +1,287 @@
+//
+//  AccountManager.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 10/24/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+struct HistoryEntry {
+
+    var description: String = ""
+    var timestamp: String = ""
+
+}
+
+enum Stages {
+
+    case loginScreen
+    case transition
+    case loginFailed
+    case finished(sessionId: String)
+
+}
+
+//MARK: -
+//MARK: Account Settings
+
+typealias LoginInfo = (netid: String, password: String)
+
+private enum BRBAccountSettings {
+    
+    static func saveToKeychain(loginInfo: LoginInfo) {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = loginInfo.netid as AnyObject
+        keychain["password"] = loginInfo.password as AnyObject
+    }
+    
+    static func removeKeychainLoginInfo() {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = nil
+        keychain["password"] = nil
+    }
+    
+    static func loadFromKeychain() -> LoginInfo? {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        guard let netid = keychain["netid"] as? String, let password = keychain["password"] as? String else {
+            return nil
+        }
+        return (netid: netid, password: password)
+    }
+    
+}
+
+private let loginURLString = "https://get.cbord.com/cornell/full/login.php?mobileapp=1"
+private let maxTrials = 3
+private let trialDelay = 500
+
+//MARK: -
+//MARK: Account Manager
+
+protocol BRBAccountManagerDelegate {
+    func BRBAccountManagerDidFailToQueryAccount(with error: String)
+    func BRBAccountManagerDidQueryAccount(account: BRBAccount)
+}
+
+private protocol BRBConnectionDelegate {
+
+    func BRBConnectionHandlerDelegateDidRetrieveSessionId(id: String)
+    func BRBConnectionHandlerDelegateDidFailLogin(with error: String)
+
+}
+
+private class BRBConnectionHandler: WKWebView, WKNavigationDelegate {
+    
+    var stage: Stages = .loginScreen
+    private var diningHistory: [HistoryEntry] = []
+    private var loginCount = 0
+    var netid: String = ""
+    var password: String = ""
+    var delegate: BRBConnectionDelegate?
+    
+    init() {
+        super.init(frame: .zero, configuration: WKWebViewConfiguration())
+        navigationDelegate = self
+        URLCache.shared.diskCapacity = 0
+        URLCache.shared.memoryCapacity = 0
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    //MARK: -
+    //MARK: Connection Handling
+    
+    /**
+     - Gets the HTML for the current web page and runs block after loading HTML into a string
+     */
```

Comment:
Rewrite as `/// Gets the ...`. Google's Swift style guide says no to Java style comments.